src/contexts/wishlist.context.js

```diff
@@ -0,0 +1,31 @@
+import { createContext, useContext, useReducer } from "react";
+import { wishlistArr } from "../data/index.js";
+
+const WishListContext = createContext()
+
+export const WishListProvider = (props) => {
+    const [wishlistState, wishlistDispatch] = useReducer(wishListReducer, wishlistArr)
+
+    function wishListReducer(state, action) {
+
```

Comment:
Try to use either arrow functions or the function keyword. Avoid using them together

Reply:
actually, i was going by this practice where functions inside components would use `function` keyword. thank you.